private/tools/java/com/github/bazelbuild/rules_jvm_external/BUILD

```diff
@@ -1,11 +1,6 @@
 java_library(
     name = "rules_jvm_external",
     srcs = glob(["*.java"]),
-    javacopts = [
```

Comment:
We use this to keep ourselves honest. Our users should still be able to target and use Java 11 in their builds. Without this, it'd be easy for us to accidentally rely on features that aren't available until later versions of the JDK.

Reply:
I believe that produced .jar as well as source code limitations are already controlled with --java_language_version and --java_release_version. 

Demo for a source version: https://github.com/bazelbuild/rules_jvm_external/pull/1231

Manual check for class version:
```
$ bazel build //private/tools/java/com/github/bazelbuild/rules_jvm_external
$ unzip bazel-bin/private/tools/java/com/github/bazelbuild/rules_jvm_external/librules_jvm_external.jar
$ javap -verbose com/github/bazelbuild/rules_jvm_external/ByteStreams.class | grep "major"
  major version: 55
```

Reply:
But the bootclasspath differs: without https://github.com/bazelbuild/rules_java/pull/182, it's determined by `--java_runtime_version`, not `--java_language_version`. This makes it harder to e.g. test on JDK 21, but at the same time have compile-time assurance that the output is compatible with Java 11.